File: mukemmelSayı.c
#include <stdio.h>

int main()
{
    int sayı;
    int toplam = 0;

    scanf("%d",&sayı);


    for(int i=1;i<sayı;++i){
        if(sayı%i==0){
            toplam+=i;
        }
    }
    if(toplam == sayı){
        printf("Mukemmel sayıdır");
    }
    return 0;
}

File: mukemmelSayı.py
toplam = 0

sayı = int(input("Bir sayı giriniz :"))


for i in range(1,1000):
    if sayı % i == 0:
        toplam += i
        
    else:
        break
        
if toplam == sayı:
  print("Mükemmel Sayıdır")
else:
  print("Mükemmel Sayı Değildir")

